lighthouserc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
         collect: {
             staticDistDir: './.next',
             numberOfRuns: 2,
-            url: ['http://localhost/server/pages/articles.html'],
+            url: ['http://localhost:3000/server/pages/404.html'],
```

Comment:
what is this for Amanda ?

Reply:
The lighthouse tests are failing due to unrelated changes https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1652816182699-33880.report.html#seo but also because we don't have descriptions yet

Reply:
I can revert it once the description changes are done.